core/src/main/resources/lib/layout/html.jelly

```diff
@@ -155,9 +155,8 @@ ${h.initPageVariables(context)}
       </j:forEach>
     </j:if>
 
-  <j:invokeStatic var="j" className="jenkins.model.Jenkins" method="getActiveInstance" />
-    <j:set var="installState" value="${j.installState.name()}" />
-    <j:if test="${installState == 'NEW' || installState == 'INITIAL_PLUGINS_INSTALLING'}">
+    <j:invokeStatic var="j" className="jenkins.model.Jenkins" method="getInstance" />
+    <j:if test="${!j.installState.setupComplete}">
```

Comment:
@daniel-beck this is the minor change - if session timed out and you had to enter the security token again, it was only including the correct CSS and such when install state was NEW and INITIAL_PLUGINS_INSTALLING, e.g. not configuring security or creating the admin user.